packages/protobuf-bench/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "dependencies": {
     "@bufbuild/protobuf": "0.2.1",
-    "esbuild": "^0.15.10",
-    "google-protobuf": "^3.20.1"
+    "esbuild": "^0.15.12",
+    "google-protobuf": "^3.21.2"
```

Comment:
If we update the package google-protobuf to the latest version, we should also update the code generator to the latest version. The generated code depends on this package. 

The generator is pinned [here](https://github.com/bufbuild/protobuf-es/blob/33ce05a2fbe1351a271070228f3f4c447cbb5162/packages/protobuf-bench/buf.gen.yaml#L11). Can you update this?

Reply:
Thanks Timo and Steve. How do you update it? curious

Reply:
You just have to change the value in that `buf.gen.yaml` that Timo linked.  However, there is no plugin with that version published in the BSR yet.  Once something is published there, I'll post an update.  Until then, since I'm not sure when that will be, let's just remove the update to `protobuf` in this PR and we'll update both in a separate PR.

Reply:
Sounds good. Reverted the protobuf version update.